paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenUI.kt

```diff
@@ -68,45 +74,70 @@ internal fun ManageScreenUI(interactor: ManageScreenInteractor) {
 }
 
 @Composable
-private fun TrailingContent(isSelected: Boolean, isEditing: Boolean, isModifiable: Boolean, paymentMethodId: String?) {
+private fun TrailingContent(
+    isSelected: Boolean,
+    isEditing: Boolean,
+    isModifiable: Boolean,
+    paymentMethod: DisplayableSavedPaymentMethod,
+    interactor: ManageScreenInteractor,
```

Comment:
I haven't thought about this too much, but I _think_ it would probably be better to leave the interactor dependency at the top level, and make anything you need lower be a lambda.

Reply:
sounds good! Updated